wilsonChan/assignments/slidingwindow/lc53/lc53.py

```diff
@@ -1,13 +1,42 @@
-def max_sub_array_of_size_k(k, arr):
+# Problem Statement #
 
+# Given an array of positive numbers and a positive number ‘k,’ find the maximum sum of any contiguous subarray of size ‘k’.
 
-def main():
-  print("Maximum sum of a subarray of size K: " + str(max_sub_array_of_size_k(3, [2, 1, 5, 1, 3, 2])))
-  print("Maximum sum of a subarray of size K: " + str(max_sub_array_of_size_k(2, [2, 3, 4, 1, 5])))
+'''
+  #This for loop on lines 25-28 is giving me some trouble. The output only returns the
+  sum of the first window:
+      Maximum sum of a subarray of size K: 8
+      Maximum sum of a subarray of size K: 5
 
+  I'm trying to remove the previous element on the left side of the current window 
+  and add the next element of on the right side of the current window
+'''
```

Comment:
This is a great first-try at psuedo-coding. But even for parts that you understand, psuedo-code it out!